spec/models/content_item_spec.rb

```diff
@@ -1,12 +1,48 @@
-require 'rails_helper'
+require "rails_helper"
 
 RSpec.describe ContentItem, type: :model do
   it { should have_and_belong_to_many(:organisations) }
 
-  describe '#url' do
-    it 'returns a url to a content item on gov.uk' do
+  describe "#url" do
+    it "returns a url to a content item on gov.uk" do
       content_item = build(:content_item, base_path: '/api/content/item/path/1')
       expect(content_item.url).to eq('https://gov.uk/api/content/item/path/1')
     end
   end
+
+  describe "#create_or_update" do
+    let(:content_item_1) { { content_id: "first_id" } }
+    let(:content_item_2) { { content_id: "second_id" } }
+    let(:organisation) { build(:organisation, slug: "the-organisation") }
+
+    before do
+      create(:content_item, content_id: "first_id")
+    end
+
+    it "creates a content item if it does not exist" do
+      expect { ContentItem.create_or_update!(content_item_2, organisation) }.to change { ContentItem.count }.by(1)
+    end
+
+    it "updates a content item if it already exists" do
+      expect { ContentItem.create_or_update!(content_item_1, organisation) }.to change { ContentItem.count }.by(0)
```

Comment:
I would expect this test to check whether a value had been updated. You could replicate the original test `it 'updates the attributes'` (that is removed in commit 7c7ca5c63e4c4909d0c419b9e55dc66b72fc2f8a) that checks the value of the title.